CHANGELOG.md

```diff
@@ -1,5 +1,35 @@
 # Mattermost Mobile Apps Changelog
 
+## 1.31.0 Release
+- Release Date: May 16, 2020
+- Server Versions Supported: Server v5.19+ is required, Self-Signed SSL Certificates are not supported unless the user installs the CA certificate on their device
+
+### Compatibility
+ - Android operating system 7+ [is required by Google](https://android-developers.googleblog.com/2017/12/improving-app-security-and-performance.html).
+ - iPhone 5s devices and later with iOS 11+ is required.
+
+### Improvements
+ 
+### Bug Fixes
+
+#### All apps
+ - Fixed an issue where slash commands with long descriptions had their description text truncated in the slash command autocomplete.
+ - Fixed an issue where users could not swipe up to dismiss in-app push notifications.
+ - Fixed an issue where the username that created the webhook was shown on webhook posts instead of the name of the bot.
+ - Fixed an issue where posts on the same thread appeared to be from different threads since the "...commented on [Thread Title]" was shown on all posts in the thread.
+ - Fixed an issue where the system message for "Edit Channel Purpose" rendered markdown.
+ - Fixed an issue where pre-fetching of unread channels included archived channels.
```

Comment:
This should be a highlight.

Improved network reliability and channel switching time for unread channels by fetching new posts as soon as the app reconnects.